src/components/analyses/details/ArgumentTypes.js

```diff
@@ -0,0 +1,30 @@
+export default {
+    PARAM_TYPE: {
+        DOUBLE: "Double",
+        DOUBLE_SELECTION: "DoubleSelection",
+        ENV_VAR: "EnvironmentVariable",
+        FILE_FOLDER_INPUT: "FileFolderInput",
+        FILE_INPUT: "FileInput",
+        FILE_OUTPUT: "FileOutput",
+        FLAG: "Flag",
+        FOLDER_INPUT: "FolderInput",
+        FOLDER_OUTPUT: "FolderOutput",
+        INFO: "Info",
+        INPUT: "Input",
+        INTEGER: "Integer",
+        INTEGER_SELECTION: "IntegerSelection",
+        MULTIFILE_OUTPUT: "MultiFileOutput",
+        MULTIFILE_SELECTOR: "MultiFileSelector",
+        MULTILINE_TEXT: "MultiLineText",
+        NUMBER: "Number",
+        OUTPUT: "Output",
+        REFERENCE_ANNOTATION: "ReferenceAnnotation",
+        REFERENCE_GENOME: "ReferenceGenome",
+        REFERENCE_SEQUENCE: "ReferenceSequence",
+        SELECTION: "Selection",
+        TEXT: "Text",
+        TEXT_SELECTION: "TextSelection",
+        TREE_SELECTION: "TreeSelection",
+        VALUE_SELECTION: "ValueSelection",
+    },
+};
```

Comment:
I saw something similar in https://github.com/cyverse-de/sonora/pull/108. But it had some missing types that were present in the DE (https://github.com/cyverse-de/ui/blob/295a657052f10fbcb6400c9e8f02151ff7201828/de-lib/src/main/java/org/iplantc/de/client/models/apps/integration/ArgumentType.java).